esmvaltool/cmorizers/obs/download_scripts/download_merra2.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script downloads MERRA-2 data
+#
+# Instructions:
+#  - Create an Earthdata account: https://urs.earthdata.nasa.gov/
+#  - Link your Earthdata account to 'GES DISC' by following instructions given here:
+#    https://disc.gsfc.nasa.gov/earthdata-login
+#  - Setup download with wget: https://disc.gsfc.nasa.gov/data-access#mac_linux_wget
+#  - Change the destination directory specified below
+#  - Check the years
+#  - Run this script
+# 
+# Additional resources:
+#    - file specification document is available here: https://gmao.gsfc.nasa.gov/pubs/docs/Bosilovich785.pdf)
+
+
+
+# The destination for the files to be downloaded
+destination=/net/exo/landclim/PROJECTS/C3S/datadir/rawobsdir/Tier3/MERRA-2/
```

Comment:
It would be better to get this from config-user.yml (e.g., RAWOBS) or at least to have it as a command line argument.

Reply:
Hmmm. Reading from yml would be nice, but I have no idea how. Do you know?

Reply:
You can follow the example of the other download scripts (but they are in python).
Or you can give it as an input argument.
I think we should just avoid having hard-coded paths in the script.

Reply:
Ok, I went for the second option.